File: ruby/1.rb
# Project Euler #1 in Ruby, by antekone
# http://anadoxin.org/blog
#
# usage:
#
# $ ruby 1.rb

puts (0...1000).reject { |i| !(i % 3 == 0 or i % 5 == 0) }.inject(0) { |i,j| i + j }

File: kotlin/1.kt
/**
 * project euler #1 in kotlin by antekone
 *
 * usage:
 * $ kotlinc 1.kt && kotlin -cp . _1Kt
 */

fun main(args: Array<String>) {
    println(IntRange(0, 999).filter {
        it % 3 == 0 || it % 5 == 0
    }.sum())
}
